src/dotnet/commands/dotnet-restore3/Program.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.DotNet.Tools.Restore3
+{
+    public partial class Restore3Command
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            return Restore3CommandApp.Execute(args);
```

Comment:
Typically, the arg definitions happen in this Program.cs file, and the "logic" happens in the other file.  See dotnet-publish or dotnet-run for an example.

That being said, there really isn't "logic" in this command.  What do you think about collapsing these two files into a single file?  I think it would be easier to read.  We can spit them later, if more logic gets added.